Networking/Networking/Model/Order.swift

```diff
@@ -14,7 +14,7 @@ public struct Order: Decodable, GeneratedCopiable {
     ///
     /// Maps to `OrderStatus.slug`.
     ///
-    public let statusKey: String
+    public let statusKey: OrderStatusEnum
```

Comment:
nit: maybe `statusKey` could be renamed to `status` now that it is an enum? no worries if it involves lots of changes though!